src/fidesctl/api/ctl/migrations/versions/69d3ce8a040e_classification_management.py

```diff
@@ -0,0 +1,130 @@
+"""classification management
+
+Revision ID: 69d3ce8a040e
+Revises: 4fc34906c389
+Create Date: 2022-09-09 03:03:36.016135
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "69d3ce8a040e"
+down_revision = "4fc34906c389"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "cls_classification_detail",
+        sa.Column("id", sa.String(length=255), nullable=False),
```

Comment:
Look at best practices for how to make this performant and consistent with naming